RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/UDPClient.java

```diff
@@ -144,6 +151,11 @@ public static UDPClient getInstance() {
 	public void stop() {
 		getSocketToUDPServer().close();
 		getLogger().debug("UDP client stopped");
+		releaseInstance();
+	}
+	
+	private static void releaseInstance() {
+		uniqueUDPClientInstance = null;
```

Comment:
Same here, see my comment for TCPClient.java

Reply:
no problem, reorganized this.